src/views/Package/PackageDocs.tsx

```diff
@@ -67,7 +67,7 @@ export const PackageDocs: FunctionComponent = () => {
     <Grid
       columnGap={4}
       h="full"
-      templateColumns={{ base: "1fr", md: "1fr 3fr" }}
+      templateColumns={{ base: "1fr", md: "20rem 3fr" }}
```

Comment:
Could you try using `minmax(20rem, 1fr)` instead? I love the addition of setting a minimum width, but we can leverage the `minmax` function so that users with wider resolutions aren't constrained to the minimum `20rem` size

Reply:
Sure thing!